pages/posts/[slug].tsx

```diff
@@ -31,13 +64,41 @@ const PostPage = ({ postMetadata, content }: postData) => {
               <p>slug: {slug}</p>
               <p>taxonomy: {taxonomy}</p>
               <p>language: {language}</p>
+              <p>img: {imageUrl}</p>
             </header>
-            <hr />
-            <MdToHtml mdSource={content} />
+            <hr /> */}
+            <div>
+              <div className="min-h-[280px] w-full relative pl-6 pt-6 pr-24 md:rounded-3xl rounded-none sm:pr-96 sm:pt-12 flex flex-col gap-y-6 md:gap-y-8">
+                <div className="w-full h-full absolute inset-x-0 top-0 bg-black/50 z-10 md:rounded-3xl rounded-none"></div>
+                  <img src="/assets/icons/arrowBack.svg" className="relative z-20 w-4 h-5" alt="" />
+                  <Image src={image} alt={slug} fill objectFit='cover' className='md:rounded-3xl rounded-none'/>
```

Comment:
At mobile screens main image should have rounded bottom corners.

Reply:
It looks good in figma but its not pretty when combined with full layout - looks out of place, thats why i have made the image rounded only on tablet and up